package.json

```
@@ -138,7 +138,7 @@
 		"translate:calypso": "wp-babel-makepot './{client,packages,apps}/**/*.{js,jsx,ts,tsx}' --ignore '**/node_modules/**,**/test/**,**/*.d.ts' --base './' --dir './build/strings' --output './public/calypso-strings.pot'",
 		"tsc": "NODE_OPTIONS='--max-old-space-size=4096' tsc",
 		"typecheck": "yarn run tsc --project client",
-		"update-deps": "rm -rf yarn.lock && yarn run distclean && yarn install && replace --silent 'http://' 'https://' . --recursive --include='yarn.lock' && touch -m node_modules",
+		"update-deps": "rm -rf yarn.lock && yarn run distclean && yarn install && sed -i '' 's|http://|https://|g' yarn.lock && touch -m node_modules",
```

`sed -i ""` will be only for macOS right? Should we also add support for Linux, just in case (without the quotes)?

Could be something like this (untested):
```suggestion
		"update-deps": "rm -rf yarn.lock && yarn run distclean && yarn install && (sed -i '' 's|http://|https://|g' yarn.lock || sed -i 's|http://|https://|g' yarn.lock) && touch -m node_modules"
```

---

> `sed -i ""` will be only for macOS right? Should we also add support for Linux, just in case (without the quotes)?

That's a good flag. I would expect this command would be most likely to be run on a developer's own computer and Linux being less common than macOS, but I think the compatibility is worth it.

After researching a bit more, I think this could be substituted with a `perl` command that works across both Linux and macOS

```suggestion
		"update-deps": "rm -rf yarn.lock && yarn run distclean && yarn install && perl -pi -e s,http://,https://,g yarn.lock && touch -m node_modules",
```

Or use `-i` with a output file copy and replace the original

```suggestion
		"update-deps": "rm -rf yarn.lock && yarn run distclean && yarn install && sed -i.bak '' 's|http://|https://|g' yarn.lock && mv yarn.lock.bak yarn.lock && touch -m node_modules",
```

---

I pushed the `perl` alternative in 4db441a.